managed/ui/src/components/certificates/Certificates.js

```diff
@@ -153,32 +208,26 @@ class Certificates extends Component {
         >
           <i className="fa fa-download"></i> Download Root CA Cert
         </MenuItem>
+        <MenuItem
+          onClick={() => { 
+            !deleteDisabled && this.deleteRootCertificate(payload?.uuid)
+          }}
+          disabled={deleteDisabled}
+        >
+          <i className="fa fa-trash"></i> Delete Root CA Cert
```

Comment:
it's generic cert delete, so `root CA` shouldn't be in the label

Reply:
Done